space.nouspiro.tenmon.json

```
@@ -0,0 +1,133 @@
+				"-DBUILD_opencv_calib3d=OFF",
+				"-DBUILD_opencv_dnn=OFF",
+				"-DBUILD_opencv_features2d=OFF",
+				"-DBUILD_opencv_flann=OFF",
+				"-DBUILD_opencv_gapi=OFF",
+				"-DBUILD_opencv_highgui=OFF",
+				"-DBUILD_opencv_imgcodecs=OFF",
+				"-DBUILD_opencv_java_bindings_generator=OFF",
+				"-DBUILD_opencv_js:BOOL=OFF",
+				"-DBUILD_opencv_js_bindings_generator=OFF",
+				"-DBUILD_opencv_ml=OFF",
+				"-DBUILD_opencv_objc_bindings_generator=OFF",
+				"-DBUILD_opencv_objdetect=OFF",
+				"-DBUILD_opencv_photo=OFF",
+				"-DBUILD_opencv_python_bindings_generator=OFF",
+				"-DBUILD_opencv_python_tests=OFF",
+				"-DBUILD_opencv_stitching=OFF",
+				"-DBUILD_opencv_ts=OFF",
+				"-DBUILD_opencv_video=OFF",
+				"-DBUILD_opencv_videoio=OFF",
+				"-DBUILD_opencv_world:BOOL=OFF"
+			],
+			"builddir": true,
+			"sources": [
+				{
+					"type": "archive",
+					"url":"https://github.com/opencv/opencv/archive/4.5.5.zip",
+					"sha256":"fb16b734db3a28e5119d513bd7c61ef417edf3756165dc6259519bb9d23d04e2"
+				}
+			]
+		},
+		{
+			"name": "libgsl",
+			"buildsystem": "autotools",
+			"cleanup": ["/bin"],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://quantum-mirror.hu/mirrors/pub/gnu/gsl/gsl-2.7.1.tar.gz",
+					"sha256": "dcb0fbd43048832b757ff9942691a8dd70026d5da0ff85601e52687f6deeb34b"
+				}
+			]
+		},
+		{
+			"name": "cfitsio",
+			"buildsystem": "cmake",
+			"builddir": true,
+			"cleanup": ["/bin"],
+			"config-opts": [
+				"-DCMAKE_BUILD_TYPE=Release",
+				"-DUSE_PTHREADS=ON"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-4.1.0.tar.gz",
+					"sha256": "b367c695d2831958e7166921c3b356d5dfa51b1ecee505b97416ba39d1b6c17a"
+				}
+			]
+		},
+		{
+			"name": "libraw",
+			"buildsystem": "simple",
+			"cleanup": ["/bin"],
+			"build-commands": ["autoreconf -f -i", "./configure --prefix=/app", "make -j 8", "make install"],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.libraw.org/data/LibRaw-0.20.2.tar.gz",
+					"sha256": "dc1b486c2003435733043e4e05273477326e51c3ea554c6864a4eafaff1004a6"
+				}
+			]
+		},
+		{
+			"name": "wcs",
+			"buildsystem": "simple",
+			"cleanup": ["/bin"],
+			"build-commands": ["./configure --prefix=/app", "make -j 8", "make install" ],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "ftp://ftp.atnf.csiro.au/pub/software/wcslib/wcslib-7.11.tar.bz2",
+					"sha256": "46befbfdf50cd4953896676a7d570094dc7661e2ae9677b092e7fb13cee3da5f"
+				}
+			]
+		},
+		{
+			"name": "tenmon",
+			"buildsystem": "cmake",
+			"config-opts": ["-DCMAKE_BUILD_TYPE=Release", "-DRELEASE_BUILD=ON"],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://gitea.nouspiro.space/nou/tenmon.git",
+					"tag": "20221121",
+					"commit": "a7dc942c62ac82089bac66c2a2b13921bfb9976e"
+				}
+			]
+		}
+	],
+
+	"finish-args":
+	[
+		"--share=ipc",
+		"--socket=x11",
```

```suggestion
		"--socket=fallback-x11",
```

assuming this support wayland.

---

There is nasty bug with QDockWidget https://bugreports.qt.io/browse/QTBUG-87332 that is why I added `--nosocket=wayland` but otherwise it can run on Wayland.

---

I didn't notice it was nosocket. Which is useless here.

---

Qt wayland support is sketchy at best so I totally understand.